pontoon/contributors/static/css/settings.css

```diff
@@ -1,25 +1,35 @@
-#homepage {
+#homepage,
+#preferred-locale {
   margin-top: 30px;
 }
 
-#homepage .label {
+#homepage .label,
+#preferred-locale .label {
```

Comment:
What if you simply use `#heading X` instead of `#homepage X, #preferred-locale` here and below?